### PR TITLE
MxTimer matching

### PR DESCRIPTION
**Pull request by ItzSwirlz**
I'll do more, just started with noticing the call order mismatch

7:10 PM EDT: all good!

### CODE REVIEW

**Comment by WamWooWam** [comment]
how in gods name did i manage to get these backwards

**Comment by ItzSwirlz** [comment]
> how in gods name did i manage to get these backwards 

We're human. Iirc I checked again with the other way around and got this to be correct but it's late now, if someone can check the order doesn't matter that'd be awesome, but definitely the GetRealTime was confused with the other function. You might have looked at the wrong line

File: LEGO1/mxtimer.cpp [hunk 1]
```diff
@@ -10,17 +10,18 @@ long MxTimer::s_LastTimeTimerStarted = 0;
...
 MxTimer::MxTimer()
-{
-  this->m_isRunning = MX_FALSE;
-  MxTimer::s_LastTimeCalculated = timeGetTime();
-  this->m_startTime = MxTimer::s_LastTimeCalculated;
+{
+  this->m_isRunning = MX_FALSE;
+  m_startTime = timeGetTime();
+  // yeah this is somehow what the asm is
+  s_LastTimeCalculated = m_startTime;
```
Comment (madebr) [inline LEGO1/mxtimer.cpp:17]: Does the following generate the same asm?
This looks like something that the original devs could have written.
```c++
  s_LastTimeCalculated = m_startTime = timeGetTime();

```

**Comment by itsmattkc** [comment]
Looks good, thanks!

File: LEGO1/mxtimer.cpp (hunk 1 above)
Comment (ItzSwirlz) [reply LEGO1/mxtimer.cpp:17]: While trying to generate a 100% match I noticed some comment (?) made it match

**Pull request merged**